tests/unit/bokeh/embed/test_bundle.py

```diff
@@ -122,6 +122,13 @@ def test_env_vars_precedence(self) -> None:
         # this is a cheap test that a BokehJS file is NOT inline
         assert all(len(x) < 5000 for x in b.js_raw)
 
+    def test_no_objs_all_resources_bundled(self) -> None:
+        b = beb.bundle_for_objs_and_resources(None, 'relative')
```

Comment:
Looks like this causes issues on windows. It seems like a real issue that we may need to handle, but for the purpose of this test `'absolute'` resources will be just fine to clear the CI.

Reply:
Oh, I see. Thanks for figuring that out, fixed now.